sycl/test/xocc_tests/simple_tests/ddr_bank_test.cpp

```diff
@@ -0,0 +1,40 @@
+// REQUIRES: xocc
+
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out 2>&1 | FileCheck %s
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+
+#include <CL/sycl.hpp>
+
+int main() {
+  cl::sycl::buffer<cl::sycl::cl_int, 1> Buffer(4);
+  cl::sycl::queue Queue;
+  cl::sycl::range<1> NumOfWorkItems{Buffer.get_count()};
+
+  Queue.submit([&](cl::sycl::handler &cgh) {
+    sycl::ONEAPI::accessor_property_list PL{sycl::xilinx::ddr_bank<1>};
```

Comment:
Remove this intermediate variables.
Rationale: they are going to change their type soon and there will be a merge conflict.